.github/workflows/unit_tests.yml

```diff
@@ -8,6 +8,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [ 3.7, 3.8, 3.9 ]
+        go-version: [ 1.17.7 ]
```

Comment:
I dont think we need a matrix. Would we ever need to use more than one version?

Reply:
isn't there a possibility we would want to support e.g. the last 2 minor versions of Go? when 1.18 is released, would we not want to continue supporting 1.17?

Reply:
Do you mean for our build process or for developers? The only breaking change will be to 2.0 as far as I can tell.